IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormPullTests.cs

```diff
@@ -207,6 +207,27 @@ public void Should_remember_user_choice_upon_pull(bool autoStashChecked)
                 AppSettings.PullAction.Merge);
         }
 
+        [Test]
+        public void Should_disable_enable_merge_options_when_no_tracking()
+        {
+            RunFormTest(
+                form =>
+                {
+                    var accessor = form.GetTestAccessor();
+
+                    accessor.DisableMergeOptionsWhenCurrentBranchIsNotTracking();
```

Comment:
What's state of Merge, Rebase, Fetch controls before `DisableMergeOptionsWhenCurrentBranchIsNotTracking` is run? Is their state expected?

Reply:
Since we're using **_commands.StartPullDialog**, later **FormPullLoad** gets called and in it we call  **DisableMergeOptionsWhenCurrentBranchIsNotTracking()** , so yes, no need to call it again after the accessor initialization.

Thus, states of the controls are:
Rebase.Enabled = false
Merge.Enabled  = false
Fetch:.Enabled   = true
Fetch.Checked = true

(Also fixed !isLocalBranch in if statement because Fetch.Checked was set to true when the branch is **not** the local one, if it's available in remote, we should set Merge to true, not Fetch)

Reply:
> Since we're using **_commands.StartPullDialog**, later **FormPullLoad** gets called and in it we call **DisableMergeOptionsWhenCurrentBranchIsNotTracking()** , so yes, no need to call it again after the accessor initialization.
> 
> Thus, states of the controls are:
> Rebase.Enabled = false
> Merge.Enabled = false
> Fetch:.Enabled = true
> Fetch.Checked = true

Please add this as a comment before line:216, so we don't have to scratch our heads next time we look at the test.
Thank you

Reply:
Fixed, thank you for your comments 👍